docs/tutorials/dronecan_flight_controller.rst

```diff
@@ -101,6 +101,9 @@ properly configured and disarmed. So the default settings on Vertiq modules shou
 **Because of this, there is no need to change any arming parmeters to complete a basic integration with a PX4 or ArduPilot flight controller.** If you wish to take advantage of these arming
 features for more complex integrations, refer to the :ref:`Advanced Arming <manual_advanced_arming>` section for more details.
 
+If your module is set to :ref:`use ArmingStatus to drive module arming <dronecan_arming_and_bypass>`, the module will arm 
```

Comment:
I mentioned it in the other place as well, but people mess it up so much I want to drive it home. Can you mention here, either in a note, or warning, or bold, or just text with this, that you have to set up their Manual Arming Throttle Source, most likely to DroneCAN, if they are using this. I want it to be really in their faces.